### PR TITLE
Bump to 2.4.1

### PR DESCRIPTION
**Pull request by marekkirejczyk**

### CODE REVIEW

File: waffle-provider/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@ethereum-waffle/provider",
   "description": "A mock provider for your blockchain testing needs.",
-  "version": "2.4.0",
+  "version": "2.4.1",
```
Comment (sz-piotr) [inline waffle-provider/package.json:4]: No changes were made to the provider I think

File: docs/release-notes/2.4.1.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## Changes:
```
Comment (sz-piotr) [inline docs/release-notes/2.4.1.md:1]: Also mention fixing the bug in big number chai matchers:
previously `to.be.gt` didn't work
also `expect(3).to.be.at.least(utils.bigNumberify(2))` didn't pass

Comment (sz-piotr) [reply docs/release-notes/2.4.1.md:1]: And this applies to all operations that take the order of arguments into account, so `>` `>=` `<` and `<=`

File: waffle-provider/package.json (hunk 1 above)
Comment (marekkirejczyk) [reply waffle-provider/package.json:4]: Yeah, it easier for me to just bump all (need to add automatic bumping).

**Pull request merged**